common/app/views/fragments/containers/facia_cards/videoContainer.scala.html

```diff
@@ -48,7 +48,7 @@ <h2 class="video-title fc-container__header__title">
                             autoPlay = false,
                             showControlsAtStart = false
                         )) { player =>
-                            <div class="fc-item__media-wrapper u-faux-block-link__promote media__container--hidden js-video-player">
+                            <div class="fc-item__media-wrapper u-faux-block-link__promote media__container--hidden js-video-player video-playlist__item__player">
```

Comment:
nope - I branched off the other PR and squashed the commits... missed this change!